### PR TITLE
fixes #4320

### LINKED ISSUE #4320: mat-tabs-theme() doesn't apply a mat-dark-theme() font color
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
If you style only certain component as described in the doc, mat-dark-theme() doesn't apply white font color on tab headers.

#### What is the current behavior?
Tab headers have black font color

#### What are the steps to reproduce?

Here is my theme.scss (WIP)

```
@import '~@angular/material/_theming.scss';

@include mat-core();
$primary: mat-palette($mat-cyan, 700, A100, A400);
$accent:  mat-palette($mat-red, 700, A100, A400);
$warn:    mat-palette($mat-orange);

$dark-theme : mat-dark-theme($primary, $accent, $warn);
$light-theme : mat-light-theme($primary, $accent, $warn);

//@include angular-material-theme($dark-theme);
@include mat-core-theme($dark-theme);
@include mat-autocomplete-theme($light-theme);
@include mat-button-theme($light-theme);
@include mat-button-toggle-theme($light-theme);
@include mat-card-theme($light-theme);
@include mat-checkbox-theme($light-theme);
@include mat-chips-theme($light-theme);
@include mat-dialog-theme($light-theme);
@include mat-grid-list-theme($light-theme);
@include mat-icon-theme($light-theme);
@include mat-input-theme($light-theme);
@include mat-list-theme($dark-theme);
@include mat-menu-theme($light-theme);
@include mat-progress-bar-theme($light-theme);
@include mat-progress-spinner-theme($light-theme);
@include mat-radio-theme($light-theme);
@include mat-select-theme($light-theme);
@include mat-sidenav-theme($light-theme);
@include mat-slide-toggle-theme($light-theme);
@include mat-slider-theme($light-theme);
@include mat-tabs-theme($dark-theme);
@include mat-toolbar-theme($light-theme);
@include mat-tooltip-theme($light-theme);
```

#### What is the use-case or motivation for changing an existing behavior?
Using a dark background image requires to use some mat-dark-theme() components and some mat-light-theme() components.

#### Which versions of Angular, Material, OS, browsers are affected?
Angular 4.1
Material 2.0.0-beta3

#### Is there anything else we should know?
Nope :)

### PR DESCRIPTION
**Pull request by jdell64**
Made the fixes in #4320

### CODE REVIEW

**Comment by jdell64** [comment]
This is the first time I've tried to contribute, so I am not sure how to update the tests.

**Comment by willshowell** [comment]
The screenshot tests are sensitive to even those slight changes in color. AFAIK one of the team members can bypass the failure to say that the change was intentional and correct.

File: src/lib/tabs/_tabs-theme.scss [hunk 1]
```diff
@@ -31,7 +31,7 @@
   }
 
   .mat-tab-label, .mat-tab-link {
-    color: currentColor;
+     color: mat-color($foreground, text);
```
Comment (andrewseguin) [inline src/lib/tabs/_tabs-theme.scss]: nit: remove this space here

Comment (andrewseguin) [inline src/lib/tabs/_tabs-theme.scss]: Still got the space in there ;) You removed a space from the wrong spot

**Comment by jdell64** [comment]
I hope I got it this time lol

**Pull request merged**